apps/web/pages/v2/event-types/index.tsx

```diff
@@ -0,0 +1,640 @@
+import { UserPlan } from "@prisma/client";
+import { Trans } from "next-i18next";
+import Head from "next/head";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import React, { Fragment, useEffect, useState } from "react";
+
+import { CAL_URL, WEBAPP_URL } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { inferQueryOutput, trpc } from "@calcom/trpc/react";
+import { Icon } from "@calcom/ui";
+import { Alert } from "@calcom/ui/Alert";
+import Badge from "@calcom/ui/Badge";
+import ConfirmationDialogContent from "@calcom/ui/ConfirmationDialogContent";
+import { Dialog } from "@calcom/ui/Dialog";
+import EmptyScreen from "@calcom/ui/EmptyScreen";
+import { Button, Tooltip, Switch, showToast } from "@calcom/ui/v2";
+import Dropdown, {
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from "@calcom/ui/v2/core/Dropdown";
+import Shell from "@calcom/ui/v2/core/Shell";
+import CreateEventTypeButton from "@calcom/ui/v2/modules/event-types/CreateEventType";
+
+import { withQuery } from "@lib/QueryCell";
+import classNames from "@lib/classNames";
+import { HttpError } from "@lib/core/http/error";
+
+import { EmbedButton, EmbedDialog } from "@components/Embed";
+import EventTypeDescription from "@components/eventtype/EventTypeDescription";
```

Comment:
We don't use the v2 component for EventTypeDescription here. @sean-brydon is that on purpose?